Lunch/TodaysLunchViewController.swift

```diff
@@ -7,20 +7,150 @@
+
+            if let placeLikelihoodList = placeLikelihoodList {
+                let place = placeLikelihoodList.likelihoods.first?.place
+                if let place = place {
+                    var name = place.name
+                    var location = place.coordinate
+                    let span = MKCoordinateSpanMake(0.05, 0.05)
+                    let region = MKCoordinateRegion(center: location, span: span)
+                    var request = MKLocalSearchRequest()
+                    request.region = MKCoordinateRegion(center: location, span: span)
+                    request.naturalLanguageQuery = self.textForLunchLabel
+                    var search = MKLocalSearch(request: request)
+                    search.startWithCompletionHandler({ (response: MKLocalSearchResponse!, error: NSError?) -> Void in
+                        if response != nil && error == nil {
+                            var mapItems = response.mapItems as? [MKMapItem]
+                            var annotationList = [MKPointAnnotation]()
+                            for item in mapItems! {
+                                println("name \(item.name)")
+                                var annotation = CustomPointAnnotation()
+                                annotation.coordinate = item.placemark.coordinate
+                                annotation.title = item.name
+                                annotation.subtitle = item.phoneNumber
+                                annotationList.append(annotation)
+                                self.mapView.addAnnotation(annotation)
+                            }
+                            self.mapView.showAnnotations(
+                            annotationList, animated: true)
+                        }
+
+                    })
+                }
+            }
+        })
     }
-    
+
+    override func viewDidLayoutSubviews() {
+
+    }
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        selectedPlaceLabel.text = textForLunchLabel
+        selectedVotesLabel.text = "Votes: " + textForVotesLabel
+    }
+
+    override func viewWillDisappear(animated: Bool) {
+        super.viewWillDisappear(animated)
+        let vc = self.navigationController?.topViewController as? ViewController
+        vc?.votes[selectedIndex] = textForVotesLabel.toInt()!
+        vc?.updateTableView()
+    }
+
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
     }
-    
-}
+
+    func mapView(mapView: MKMapView!, viewForAnnotation annotation: MKAnnotation!) -> MKAnnotationView! {
+        if !(annotation is CustomPointAnnotation) {
+            return nil
+        }
+
+        let reuseId = "test"
+
+        var anView = mapView.dequeueReusableAnnotationViewWithIdentifier(reuseId)
+        if anView == nil {
+            anView = MKAnnotationView(annotation: annotation, reuseIdentifier: reuseId)
+            anView.canShowCallout = true
+        }
```

Comment:
I like to put the else on the same line and I think that's what I see in the Swift book too `} else {`